anaconda_updates/anaconda_updates/releases/f34.py

```diff
@@ -0,0 +1,10 @@
+from . import GeneralBranch, Branch
+
+
+class Fedora34Branch(GeneralBranch):
+
+    def __init__(self):
+        super().__init__(branch_type=Branch.fedora34,
+                         cmd_args=["-f34", "--fedora34"],
+                         help="working on Fedora 34",
+                         version_script_params=["-f34", "-p"])
```

Comment:
Please, add `site_packages="./usr/lib/python3.9/site-packages/"`.

Reply:
Thank you! Added.